.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+
+  lint-dockerfile:
+    name: Lint Dockerfile
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+      - name: Run Hadolint Dockerfile Linter
+        uses: burdzwastaken/hadolint-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          HADOLINT_ACTION_DOCKERFILE_FOLDER: .
+
+  unit-test:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+      - name: Use Java 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
```

Comment:
In travis we have:

```
jdk:
  - openjdk8
  - oraclejdk8
  - oraclejdk9
  - openjdk10
  - openjdk11
```

so I understand we should include here also versions 9, 10 and 11 (maybe using `matrix` as I've seen in PRs for nodejs repos to cover node 10 and 12?)

Reply:
Yes, you should introduce a matrix build when you are ready.